bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestFilters.java

```diff
@@ -2118,6 +2118,43 @@ public void testFuzzyWithIntKeys() throws Exception {
     assertEquals(expectedKeys, actualKeys);
   }
 
+  @Test
+  public void testRowKeysWithRegEx() throws Exception{
+    Table table = getDefaultTable();
+    String[] rowKeys = {
+            "/firstKey=1/secKey=AB/thirdKey=111",
+            "/firstKey=1/secKey=ABC/thirdKey=222",
+            "/firstKey=2/secKey=CD/thirdKey=333",
+            "/firstKey=3/secKey=BT/thirdKey=444",
+            "/firstKey=4/secKey=BTL/thirdKey=555",
+            "/firstKey=4/secKey=AD/thirdKey=666",
+            "/firstKey=5/secKey=SET/thirdKey=777",
+            "/firstKey=11/secKey=AA/thirdKey=888"
+        };
+
+    List<Put> puts = new ArrayList<>();
+    byte[] qualA = dataHelper.randomData("test-regex");
+    for (int i = 0; i < rowKeys.length; i++) {
+      String indexStr = String.valueOf(i);
+      puts.add(
+          new Put(rowKeys[i].getBytes()).addColumn(COLUMN_FAMILY, qualA, Bytes.toBytes(indexStr)));
+    }
+    table.put(puts);
+
+    // Expected rows should be sorted.
+    int[] expected = { 5, 4 };
```

Comment:
shouldn't this be `{4, 5}` ?

Reply:
The test dataset I created was not sorted So returned output is in sorted order.
I would change the order of test data as it may create confusion.

Reply:
oh... I see.  Thanks for the clarification.